Source/Icebreaker/Helpers/IcebreakerBotDataProvider.cs

```diff
@@ -248,6 +302,14 @@ private async Task InitializeAsync()
                 }
             }
 
+            // Get a reference to the Pairs collection, creating it if needed
+            var pairCollectionDefinition = new DocumentCollection
+            {
+                Id = pairsCollectionName,
+            };
+            pairCollectionDefinition.PartitionKey.Paths.Add("/id");
```

Comment:
What sets the Id in the document?

Reply:
I believe Cosmos is just assigning a random id to each PairInfo.